.github/workflows/attackbox.yml

```diff
@@ -8,30 +8,54 @@ on:
   workflow_dispatch:
     branches: [ main ]
 
+defaults:
+  run:
+    working-directory: attackbox
+
 jobs:
 
   build:
 
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout Code
+    - name: Checkout
       uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
+
     - name: Set up QEMU
       uses: docker/setup-qemu-action@v1
-    - name: Setup Docker Buildx
+
+    - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
-    - name: Login to Docker Hub
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: Login to DockerHub
```

Comment:
Same comment here, let's get rid of automatic pushes to dockerhub, given our conversation

Reply:
Done. All references to dockerhub have been scrubbed